# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../metamail/65_metamail-2.7-usesox.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mm2.7/src/bin/showaudio.usesox	Thu Feb  1 12:31:32 2001
+++ mm2.7/src/bin/showaudio	Thu Feb  1 12:36:24 2001
@@ -151,6 +151,9 @@
 #  Solaris 2.0
     /bin/audioconvert -i raw,voice -f sun $* | /bin/audioplay
     exit $status
+else if (-e /usr/bin/play) then
+# Red Hat Linux
+    exec /usr/bin/play $*
 else
     set dev=/dev/audio
 endif
